cranelift/filetests/filetests/isa/x86/simd-arithmetic-binemit.clif

```diff
@@ -176,25 +176,25 @@ block0:
 
 function %float_arithmetic_f32x4(f32x4, f32x4) {
 block0(v0: f32x4 [%xmm3], v1: f32x4 [%xmm5]):
-[-, %xmm3]    v2 = fadd v0, v1      ; bin: 40 0f 58 dd
-[-, %xmm3]    v3 = fsub v0, v1      ; bin: 40 0f 5c dd
-[-, %xmm3]    v4 = fmul v0, v1      ; bin: 40 0f 59 dd
-[-, %xmm3]    v5 = fdiv v0, v1      ; bin: 40 0f 5e dd
-[-, %xmm3]    v6 = fmin v0, v1      ; bin: 40 0f 5d dd
-[-, %xmm3]    v7 = fmax v0, v1      ; bin: 40 0f 5f dd
-[-, %xmm3]    v8 = sqrt v0          ; bin: 40 0f 51 db
+[-, %xmm3]    v2 = fadd v0, v1      ; bin: 0f 58 dd
```

Comment:
If that's not too much hassle, can you, for each different recipe involved, duplicate the test with a register that would require a REX prefix, please?

Reply:
Ok, I added some REX-inducing tests; it's not 100% coverage but that is a rather high bar with all of the variations of these instructions.